openfisca_uk/variables/finance/benefit/family/JSA_income.py

```diff
@@ -115,6 +115,10 @@ def formula(benunit, period, parameters):
             "property_income",
             "pension_income",
         ]
+        if parameters(
```

Comment:
Aesthetic - the indentation mix looks awkward to me (below too please)
```suggestion
        bi = parameters(period).contrib.ubi_center.basic_income
        if bi.include_in_means_tests:
```